projects/packages/jetpack-mu-wpcom/src/features/wpcom-colourlovers-deprecate/wpcom-colourlovers-deprecate.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Enqueue styles and scripts to display a warning message in the WordPress Customizer
+ * if a theme background image is set to a deprecated ColourLovers URL.
+ *
+ * @package automattic/jetpack-mu-plugins
+ */
+
+/**
+ * Hooks into `customize_controls_enqueue_scripts` to inject a warning message.
+ */
+function wpcom_colourlovers_deprecate_warning() {
+	// Retrieve all theme mods (customizations) set for the current theme.
+	$theme_mods = get_theme_mods();
+
+	// If no background image is set, there’s no need to display a warning.
+	if ( ! isset( $theme_mods['background_image'] ) ) {
+		return;
+	}
+
+	// Check if the background image URL contains a deprecated ColourLovers URL.
+	// If it doesn’t, we don’t need to show any warning.
+	if ( ! str_contains( $theme_mods['background_image'], 'colourlovers-static-replica.s3.amazonaws.com' ) ||
+		! str_contains( $theme_mods['background_image'], 'colourlovers.com.s3.amazonaws.com' ) ) {
+		return;
+	}
+
+	// This message is shown to users in the Customizer.
+	// It warns them that their current background image will no longer be available.
+	$message = __( 'Update your background image before April 15 to keep your site looking great.', 'jetpack-mu-wpcom' );
```

Comment:
Small nitpick to make it consistent with the notification message added in 172155-ghe-Automattic/wpcom

```suggestion
	$message = __( 'Update your background image before April 15th to keep your site looking great.', 'jetpack-mu-wpcom' );
```